collect/stressRelief.go

```diff
@@ -268,11 +270,13 @@ func (s *StressRelief) Recalc() {
 
 	var level float64
 	var reason string
+	var formula string
 	for _, c := range s.calcs {
 		stress := 100 * s.algorithms[c.Algorithm](c.Numerator, c.Denominator)
 		if stress > level {
 			level = stress
 			reason = c.Reason
+			formula = fmt.Sprintf("%s(%v/%v)=%v", c.Algorithm, c.Numerator, c.Denominator, stress)
 		}
 	}
 	s.Logger.Debug().WithField("stress_level", level).WithField("reason", reason).Logf("calculated stress level")
```

Comment:
I think we should add formula to this log message too, so you can see it with debug on, even if you're not actually in Monitor mode.

Reply:
Good call! will push that in a moment